aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/strategy/sampling/pollers/RulePoller.java

```diff
@@ -71,7 +71,7 @@ public void start() {
             try {
                 pollRule();
             } catch (Throwable t) {
-                logger.error("Encountered error polling GetSamplingRules: ", t);
+                logger.warn("Encountered error polling GetSamplingRules: ", t);
```

Comment:
I think in practice error and warn are the same, probably should switch to info.

Reply:
The key difference is that warn is not emitted at the SDK's default log level

Reply:
I don't think SDKs ever have a default log level - it's in the end user app usually in a logback.xml. It is fairly standard practice to only log things above info level I think. This is because many libraries just log all failures as warning and don't use error at all. I guess switching to warn doesn't practically change the chance of being annoyed by the log mmessages for many apps

Reply:
Ok that makes sense - I guess I'm getting my languages confused with default log levels. Went ahead and changed to info.

Reply:
hi! willarmiros! When will the latest version of the log level reduction for sampling error logs be released

Reply:
@yijiang-song unfortunately we cannot provide ETAs for our releases, please feel free to watch the repo for the next release.

Reply:
Thank you! I will continue to pay attention to this issue.